pow/src/main/java/tech/pegasys/teku/pow/Eth1Provider.java

```diff
@@ -52,4 +55,7 @@
   SafeFuture<BigInteger> getChainId();
 
   SafeFuture<Boolean> ethSyncing();
+
+  @SuppressWarnings("rawtypes")
+  SafeFuture<List<LogResult>> ethGetLogs(EthFilter ethFilter);
```

Comment:
Can we use `EthLog.LogResult<?>` here to avoid needing to suppress `rawtypes`?

Reply:
It was my first version but then i was having problem on the actual call. I give it another try.

Reply:
@ajsutton the point is that `EthLog.getLogs` returns a raw type but then the `LogResult` interface is `EthLog.LogResult<T>`.
Any suggestion here?

Reply:
Bleh. That's annoying. I'd cast it to <?> where we first get the response. Then at least we only have to suppress warnings in one place.

Reply:
```
 error: incompatible types: List<LogResult> cannot be converted to List<LogResult<?>>
              return (List<LogResult<?>>)logs;
                                         ^
```

@ajsutton 🤔

Reply:
```return (List<EthLog.LogResult<?>>)(List)logs;``` double-cast is the key

Reply:
oh my that's horrible but at least the horrible-ness is isolated to one place. Glad you found how to do it.